bin/plugin/commands/packages.js

```diff
@@ -237,8 +239,46 @@ async function updatePackages(
 		true,
 		abortMessage
 	);
-	git.commit( gitWorkingDirectoryPath, 'Update changelog files', [ './*' ] );
+	const commitHash = await git.commit(
+		gitWorkingDirectoryPath,
+		'Update changelog files',
+		[ './*' ]
+	);
 	log( '>> Changelog files changes have been committed successfully.' );
```

Comment:
Now that we know the commit hash, could we backport it to the `master` branch?

Reply:
I preferred to leave it to a manual process because most of the time, it's going to conflict.

Reply:
True, it’s fine to do it manually. Maybe we should print commands to run so it was easy to copy and paste.